science/eso-flames-fposs/Portfile

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+
+#PortGroup       compilers 1.0
+
+set instrument      flames
+
+name            eso-flames-fposs
+
+version         115.0.3
+revision        2
+
+categories      science
+license         GPL-2+
+maintainers     eso.org:john.pritchard
+homepage        http://www.eso.org/sci/observing/phase2/SMGuidelines/FPOSS.FLAMES.html
+description     ESO FLAMES Observation Preparation software
+long_description ESO FLAMES Observation Preparation software.
+                 
+master_sites        https://ftp.eso.org/pub/usg/fposs/
+distname            flames-fposs-${version}
+distfiles           fposs-${version}.tar.gz
```

Comment:
Is the reason for setting `distname` just so that the extracted directory name will be right? If so, it would be better to set `worksrcdir` instead, since `distname` is meant to reflect the naming of the source archive.
```
distname   fposs-${version}
worksrcdir flames-fposs-${version}
```